ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/antlr/JavaLexerTest.java

```diff
@@ -0,0 +1,115 @@
+package edu.usb.argos.ASTProcessor.antlr;
+
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.antlr.v4.runtime.Token;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Same here, remove the * and only call the necessary ones, do the rest in all your classes.

Reply:
Done